src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size;
+    private Node<K, V>[] table;
+
+    @SuppressWarnings("unchecked")
```

Comment:
Using `@SuppressWarnings("unchecked")` is generally discouraged as it suppresses important warnings. Instead, use the diamond operator with specified types, like `new Node<K,V>[DEFAULT_CAPACITY]`, to avoid the need for this annotation.